app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -3522,30 +3522,6 @@ class BrowserTabViewModelTest {
         }
     }
 
-    @Test
-    fun whenConsumeAliasThenPixelSent() {
-        whenever(mockEmailManager.getAlias()).thenReturn("alias")
-        whenever(mockEmailManager.getCohort()).thenReturn("cohort")
-        whenever(mockEmailManager.getLastUsedDate()).thenReturn("2021-01-01")
-
-        testee.usePrivateDuckAddress("", "foo@example.com")
-
-        verify(mockPixel).enqueueFire(
-            AppPixelName.EMAIL_USE_ALIAS,
-            mapOf(Pixel.PixelParameter.COHORT to "cohort", Pixel.PixelParameter.LAST_USED_DAY to "2021-01-01"),
-        )
-    }
-
-    @Test
-    fun whenCancelAutofillTooltipThenPixelSent() {
```

Comment:
Tested elsewhere, in https://github.com/duckduckgo/Android/pull/3582/files#diff-86ed09c4f784467e8f157d920ab5dbfcedbf2995ff42181622cf763407052d50